code/modules/mapfluff/ruins/spaceruin_code/listeningstation.dm

```diff
@@ -115,4 +123,17 @@
 	<br>You may view intelligence reports from your predecessors in the filing cabinet in your office.
 	"}
 
-
+// MONKESTATION EDIT: monkestation new listening post. im not making a seperate file for modularising something that already exists on tg. please god.
```

Comment:
oh if this is actually on TG then you dont need to mark this

Reply:
improperly marked: the fluff exists, this bit doesnt